modules/govuk_bouncer/manifests/gor.pp

```diff
@@ -27,13 +27,17 @@
   }
 
   class { 'govuk_gor':
-    args   => {
+    args    => {
       '-input-raw'          => ':80',
       '-output-http'        => $gor_targets,
-      '-output-http-method' => [
+      '-http-allow-method'  => [
         'GET', 'HEAD', 'OPTIONS',
       ],
+      '-http-original-host' => '',
     },
-    enable => $gor_enable,
+    envvars => {
+      'GODEBUG' => 'netdns=go',
```

Comment:
A quick google reveals that this is the default, I think? Would it make sense to simplify this manifest and remove this environment variable?